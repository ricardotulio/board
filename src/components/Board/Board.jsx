import React from 'react'
import styles from './styles.module.css'

const Board = ({ children }) => (
  <div className={styles.board}>
    <div className={styles.container}>
      <div className={styles.columns}>
        { children }
      </div>
    </div>
  </div>
)

export default Board
