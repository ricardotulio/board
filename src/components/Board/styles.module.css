.board {
  display: inline-flex;
  min-height: 100vh;
  min-width: 100vw;
  padding: 10px;
  background-color: rgb(76, 154, 255);
}

.columns {
  display: flex;
}
