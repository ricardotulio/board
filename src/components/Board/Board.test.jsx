import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import Board from './Board'

it('can render a board', () => {
  const { container } = render(<Board />)
  expect(container.firstChild).toMatchSnapshot()
})
