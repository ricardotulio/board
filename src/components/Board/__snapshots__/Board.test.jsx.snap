// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a board 1`] = `
<div
  class="board"
>
  <div
    class="container"
  >
    <div
      class="columns"
    />
  </div>
</div>
`;
